src/region_growing.jl

```diff
@@ -0,0 +1,137 @@
+
+# An implementation of the Improved Seeded Region Growing Algorithm in Julia
+#
+# Citation:
+# Albert Mehnert, Paul Jackaway (1997), "An improved seeded region growing algorithm"
+# http://www.ee.bgu.ac.il/~itzik/IP5211/Other/Projects/P19_Seeded%20region%20growing.pdf
+
+function srg{CT<:Colorant}(img::AbstractArray{CT,2}, seeds::AbstractVector{Tuple{CartesianIndex{2}, Int}}, diff_fn::Function = (c1,c2)->(sum(abs,c1-c2)))
```

Comment:
I was thinking of exporting only a single function `region_growing` which if passed with seeds would call `srg` and if passed without them, would call `urg`. `srg` and `urg` would not be exported directly. But yes, I will make the name more desciptive.